app/lib/themes.dart

```diff
@@ -10,14 +12,43 @@ class Themes {
   Themes(this.lightTheme, this.darkTheme);
 
   static Themes getNewTheme(Color seedColor) {
-    // The basic theme, global theme data changes should be put here.
     ThemeData basicTheme(Brightness brightness) {
-      return ThemeData(
-        colorScheme:
-            ColorScheme.fromSeed(seedColor: seedColor, brightness: brightness),
-        inputDecorationTheme:
-            const InputDecorationTheme(border: OutlineInputBorder()),
-      );
+      if (globalStorage.prefs.getString("theme") == "amoled") {
```

Comment:
We do not use magic strings to use the globalStorage class. Have a look at storage.dart and other calls of globalStorage